dasharo-stability/capsule-update.robot

```diff
@@ -278,3 +364,31 @@ Display Preparation Instructions
     Log To Console    Without it, a successful flash of DUT will prevent tests from working
     Log To Console    correctly.
     Log To Console    \n******************************************************************************
+
+Prepare DUT For Logo Persistence Test
+    IF    '${MANUFACTURER}' != 'QEMU'
```

Comment:
Is there a reason why it wouldn't work on QEMU, or just not tested?

Reply:
Yeah this is not a good test for whether persistent bootsplash is supported.

We could add another flag, or have the test check if it's supported:

```
$ cbfstool build/coreboot.rom layout -w | grep BOOTSPLASH > /dev/null && echo "Custom logo supported" || echo "Custom logo not supported"
Custom logo supported
```

Reply:
There is no way to flash the dut in qemu to prepare it fot the test. It can be done differently by preparing original qemu.rom file with bootsplash logo and then preparing capsule update.
This is only because you have to manualy change fw in qemu to original one after each test.